src/commons/interfaces.ts

```
@@ -32,6 +32,34 @@ export interface Projeto {
+  titulo: string
+  descricao: string
+  dataCriacao?: Date
+  publicoAlvo?: string
+  status?: string
+  aluno?: {
+    id: number | null
+    nome: string
+  }
+  professor?: {
+    id: number | null
+    nome: string 
```

ESLint

[error] Delete `·`

(prettier/prettier)